scripts/validate_docstrings.py

```diff
@@ -490,10 +491,41 @@ def yields(self):
     @property
     def method_source(self):
         try:
-            return inspect.getsource(self.obj)
+            source = inspect.getsource(self.obj)
+            # Remove common indentation.
+            first_spaces = re.match(' +', source)
+            if first_spaces:
+                first_spaces = first_spaces.group(0)
+                source = re.sub('^' + first_spaces, '', source)
+                source = re.sub(r'\n' + first_spaces, r'\n', source)
+            return source
         except TypeError:
             return ''
 
+    @property
+    def method_returns(self):
```

Comment:
I don't think it's very reusable a property that returns all possible returns of a function. I'd instead have a `method_returns_something` or something like that, returning a boolean on whether the function needs a Returns section in the docstring.

Also, please add a docstring here explaining what this does.